test/javascripts/remote_test.coffee

```diff
@@ -1,7 +1,18 @@
+baseHTML = '<html><head></head><body></body></html>'
+iframe = null
+
+setupIframe = ->
+  iframe = document.createElement('iframe')
+  document.body.appendChild(iframe)
+  iframe.contentDocument.write(baseHTML)
+  iframe.contentDocument
+
 describe 'Remote', ->
   @initiating_target = null
 
   beforeEach ->
+    testDocument = setupIframe() unless iframe
```

Comment:
Does this mean that the previous test's url/content will leak into the current test?